migration/src/lib.rs

```diff
@@ -34,6 +34,8 @@ mod m0000315_create_cvss3_scoring_function;
 mod m0000320_create_cvss3_score_column;
 mod m0000325_create_cvss3_severity_column;
 mod m0000330_add_sbom_version;
+mod m0000350_add_sbom_file;
```

Comment:
These stacked PRs that encompass previous PRs are sometimes a challenge for me to suss out which part needs reviewing.